pkg/estimator/client/cache.go

```diff
@@ -49,23 +49,13 @@ type clientWrapper struct {
 	client     estimatorservice.EstimatorClient
 }
 
-// IsEstimatorExist checks whether the cluster estimator exists in the cache.
-func (c *SchedulerEstimatorCache) IsEstimatorExist(name string) bool {
-	c.lock.RLock()
-	defer c.lock.RUnlock()
-	_, exist := c.estimator[name]
-	return exist
-}
```

Comment:
There is no need to remove this method. It is an exported function and may be used elsewhere.

Reply:
done,Thanks~